Testscripts/Linux/utils.sh

```diff
@@ -2181,22 +2186,25 @@ function remove_package () {
 function install_epel () {
 	case "$DISTRO_NAME" in
 		oracle|rhel|centos)
-			if [[ $DISTRO_VERSION =~ ^6\. ]]; then
-				epel_rpm_url="https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm"
-			elif [[ $DISTRO_VERSION =~ ^7\. ]]; then
-				epel_rpm_url="https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm"
-			elif [[ $DISTRO_VERSION =~ ^8\. ]]; then
-				epel_rpm_url="https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm"
-			else
-				LogErr "Unsupported version to install epel repository"
-				return 1
+			yum -y install epel-release
```

Comment:
why need this?

Reply:
I found there are multiple ways installing epel-release, and the original script took the Fedora releases, but CentOS has epel-release primarily, And Fedora packages do not work for now. I put this epel-release repo access as primary, and keep the original epel package installation from Fedora is secondary.  Found one article here https://www.cyberciti.biz/faq/installing-rhel-epel-repo-on-centos-redhat-7-x/